packages/material-ui/src/FormControlLabel/FormControlLabel.js

```diff
@@ -119,15 +124,15 @@ FormControlLabel.propTypes = {
    * Override or extend the styles applied to the component.
    * See [CSS API](#css) below for more details.
    */
-  classes: PropTypes.object.isRequired,
+  classes: PropTypes.object,
   /**
    * @ignore
    */
   className: PropTypes.string,
   /**
    * A control element. For instance, it can be be a `Radio`, a `Switch` or a `Checkbox`.
    */
-  control: PropTypes.element,
+  control: PropTypes.element.isRequired,
```

Comment:
This was incorrect. We call `cloneElement` on the value. The TypeScript types were correct